### PR TITLE
add simple awsim running video

### PR DESCRIPTION
**Pull request by alanmengg**
Signed-off-by: guiping meng <alan.meng@autocore.ai>

## Description

add simple awsim running example video
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by shmpwk** [comment]
cc. @mackierx111 @YoshinoriTsutake

File: docs/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial.md [hunk 1]
```diff
@@ -212,3 +212,5 @@ ros2 bag play <ROSBAG_NAME>
 If the expected ROS topic data is not produced, or the point cloud is not visible on the simulation screen, report the problem as an [Autoware Q&A discussion](https://github.com/autowarefoundation/autoware/discussions/).
 
 To make analysis of the problem easier, make sure to provide a description of your environment, a detailed set of reproduction steps, screenshots or a short video of any unexpected behaviour and attach the `Player.log` file, which can be found in the `~/.config/unity3d/Tier\ IV/E2ESimulator/Player.log` directory.
+
+[simple running video link](https://drive.google.com/file/d/1qxImNdK7d1zhMsJqhOa2NEM3vnjKSY7B/view?usp=share_link)
```
Comment (xmfcx) [inline docs/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial.md:216]:
```suggestion
![type:video](https://drive.google.com/file/d/1qxImNdK7d1zhMsJqhOa2NEM3vnjKSY7B/view?usp=share_link)
```
Following https://pypi.org/project/mkdocs-video/

cc @kenji-miyake do we have this plugin?

Comment (xmfcx) [reply docs/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial.md:216]: Created a PR in https://github.com/autowarefoundation/autoware-github-actions/pull/202 @kenji-miyake

**Comment by xmfcx** [comment]
@alanmengg

It seems with https://github.com/autowarefoundation/autoware-documentation/pull/206 the info on this documentation moved to the https://tier4.github.io/AWSIM/DeveloperGuide/Documentation/

https://autowarefoundation.github.io/autoware-documentation/main/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial/

Could you make this PR suggestion for [awsim](https://tier4.github.io/AWSIM) repository?

I think it's ok to add it to https://github.com/tier4/AWSIM/blob/main/docs/GettingStarted/QuickStartDemo/index.md

File: docs/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial.md (hunk 1 above)
Comment (kenji-miyake) [reply docs/tutorials/ad-hoc-simulation/digital-twin-simulation/awsim-tutorial.md:216]: Seeing https://github.com/squidfunk/mkdocs-material/issues/4053, it's may be possible to the HTML video tag.
@xmfcx Which do you think is better?

**Comment by alanmengg** [comment]
> https://github.com/tier4/AWSIM/blob/main/docs/GettingStarted/QuickStartDemo/index.md

ok, i will make new one

**Pull request closed without merging**